lib/web3/eth/eth_module.rb

```diff
@@ -5,47 +5,65 @@ class EthModule
 
       include Web3::Eth::Utility
 
-      PREFIX = 'eth_'
+      PREFIX = 'eth_'.freeze
 
-      def initialize web3_rpc
+      def initialize(web3_rpc)
```

Comment:
While the Ruby community as a whole is certainly not unified in its preference for parens or no parens in a method definition, @izetex's existing style in this project seems to be no parens. I think it's good form to maintain the same style as the project creator.

***

No parens is sometimes referred to as `Seattle.rb style`:

https://twitter.com/mrb_bk/status/429000440429031424